examples/diff.c

```diff
@@ -335,3 +363,27 @@ static void diff_print_stats(git_diff *diff, struct opts *o)
 	git_buf_free(&b);
 	git_diff_stats_free(stats);
 }
+
+int extract_break_rewrites(int *m, int *n, const char *arg)
+{
```

Comment:
Made all the changes mentioned in all the comments, I read all the comments once again. I pretty sure there should not be any mistakes now.